client/src/pages/private/SiteViewDetails.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Page, CheckPermissions } from '../../components/generic';
+import { Box, Typography } from '@material-ui/core';
+import { scrollUp } from '../../utils';
+import store from 'store';
+import { Link } from 'react-router-dom';
+import routes from '../../constants/routes';
+
+
+export default ({ match }) => {
```

Comment:
Nitpick: `match` is pretty ambiguous. I know we're using `id` everywhere so maybe something like `siteId` would work.

Reply:
match is the parameter provided by React, I don't have (easy) control over how it is passed.